kinetic/doc-build.yaml

```diff
@@ -0,0 +1,60 @@
+%YAML 1.1
+# ROS buildfarm doc-build file
+---
+canonical_base_url: http://docs.ros.org
+documentation_type: rosdoc_lite
+jenkins_job_priority: 89
+jenkins_job_timeout: 120
+notifications:
+  committers: false
+  emails:
+  - ros-buildfarm-kinetic@googlegroups.com
```

Comment:
This mailing list doesn't exist yet and needs to be created with the same settings / role / members as for the previous distributions.